helm/examples/azure/hpcc-azurefile/templates/create-pvc.yaml

```diff
@@ -6,9 +6,10 @@ metadata:
 spec:
   accessModes:
     - {{ $plane.rwmany | default false | ternary "ReadWriteMany" "ReadWriteOnce" }}
-  storageClassName: {{ printf "%s-%s-sc" $plane.name (include "hpcc-azurefile.fullname" $) }}
+  storageClassName: {{ include "hpcc-azurefile.SCName" (dict "root" $ "plane" $plane) | default "" | quote  }}
```

Comment:
doesn't look like it was correct before, but I don't see where "storageClass" is being picked up (e.g. as per values-auto-azurefile.yaml), so that I can point to existing storage class and use it for ephemeral storage

Reply:
Using values-auto-azurefile.yaml shouldn't utilize this hpcc-azurefile chart. Azure will pick the storageClass

Reply:
right, yes my mistake - helm/examples/azure/values-auto-azurefile.yaml , is used on it's own, fed to hpcc to use "azurefile" storage. It's been a while since I used it!